### PR TITLE
OBDS-126 - Add modal for Bin Locations import

### PR DESCRIPTION
**Pull request by DaRoTP**

### CODE REVIEW

File: src/js/components/locations-configuration/ImportBinModal.jsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+            data={this.state.supportLinks}
+            options={{ renderInnerHtml: true }}
+          />
+          <div className="dropzone-container align-self-stretch mt-3">
+            <Dropzone
+              onDrop={this.onDrop}
+              className="dropzone-content"
+            >
+              <h4 className="mb-5">
+                <Translate id="react.productsConfiguration.dragAndDrop.label" defaultMessage="Drag and drop file here." />
```
Comment (awalkowiak) [inline src/js/components/locations-configuration/ImportBinModal.jsx:127]: Use the label from locationsConfiguration

File: src/js/components/locations-configuration/ImportBinModal.jsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+              </h4>
+              <button type="button" className="btn btn-dropzone">
+                <Translate id="react.productsConfiguration.openDialog.label" defaultMessage="OPEN FILE DIALOG" />
```
Comment (awalkowiak) [inline src/js/components/locations-configuration/ImportBinModal.jsx:130]: Use the label from locationsConfiguration

File: src/js/components/locations-configuration/ImportBinModal.jsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+              </button>
+            </Dropzone>
+          </div>
+          <div className="align-self-start">
+            <span style={{ color: '#5D9531' }} className="font-weight-bold">
+              <Translate id="react.productsConfiguration.acceptedFile.label" defaultMessage="Accepted File" />:&nbsp;
```
Comment (awalkowiak) [inline src/js/components/locations-configuration/ImportBinModal.jsx:136]: Use the label from locationsConfiguration

**Review by awalkowiak — CHANGES_REQUESTED** [review]

**Comment by DaRoTP** [comment]
I created a new **FileDrop** component because there were many instances of the same chunk of code for importing files like in _ImportBinModal, ImportLocations_ and _ImportCategories_

**Pull request merged**